apps/communications/contacts/js/views/details.js

```diff
@@ -435,68 +425,18 @@ contacts.Details = (function() {
   };
 
   var renderSocial = function cd_renderSocial(contact) {
```

Comment:
I'm not sure if we want to do that. Let me explain myself. If we remove this method, share button will not be rendered so, at least for me, it will cause a bug. I know this code will be replaced soon but in the meantime we should maintain the share button, wdyt?